docker/services/efp-devnet.yml

```
@@ -0,0 +1,29 @@
+    container_name: efp-devnet
+    image: ghcr.io/ethereumfollowprotocol/contracts/devnet:master-c76225c
+    pull_policy: always
+    environment:
+      DEVNET_HOST: "0.0.0.0"
+      DEVNET_RPC_URL: "http://devnet:8545"
+      DEVNET_SCENARIO: "demoGraph"
+      DEVNET_SAVE_DEPLOYMENTS: "true"
+    volumes:
+      - efp-deployments:/app/deployments
+    healthcheck:
+      test: [ "CMD", "curl", "--fail", "-s", "http://localhost:8000/health" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)